src/main/java/gg/packetloss/grindstone/economic/store/MarketComponent.java

```diff
@@ -203,7 +203,7 @@ public void buyCmd(CommandContext args, CommandSender sender) throws CommandExce
 
         @Command(aliases = {"sell", "s"},
                 usage = "", desc = "Sell an item",
-                flags = "hau", min = 0, max = 0)
+                flags = "haui", min = 0, max = 0)
```

Comment:
Maybe this should be `-s` instead of `-i`? 

Per the Minecraft Wiki:
> The inventory consists of 4 armor slots, 27 storage slots, 9 hotbar slots, and an off-hand slot.

Seems like "inventory"/"i" could be confusing.

Reply:
Changed as per a0ea8cd626d2098eea94746682ad528224187d2a